### DISCUSSION TITLE
Cannot change my Github account link

### DISCUSSION THREAD

**Discussion by RomainBillot** [dailydotdev/daily#655] [Feature Requests]
When registering, I was on a GH account, and I chose this account while I wanted another one.

Is there a way to change this without going though the whole tags selection process?

I could be great to manage OAuth integration (link/unlink) somewhere

**Comment by idoshamun** [comment]
Hi @RomainBillot,

Unfortunately, there is no way at the moment to manage the social providers.
All is not lost, we're eagerly working on releasing this feature and you should expect to see it in the upcoming weeks.

Best,
Ido

**Reply by idoshamun** [reply]
I'm glad to announce that now you can connect multiple accounts and even unlink some.
Please let me know once you try that
